ginga/cmap.py

```diff
@@ -11421,55 +11427,76 @@
 class ColorMapError(Exception):
     pass
 
+
 class ColorMap(object):
+    """Class to handle color maps."""
     def __init__(self, name, clst):
         self.name = name
         self.clst = clst
 
+
 def add_cmap(name, clst):
+    """Add a color map."""
     global cmaps
     assert len(clst) == min_cmap_len, \
            ValueError("color map '%s' length mismatch %d != %d (needed)" % (
         name, len(clst), min_cmap_len))
     cmaps[name] = ColorMap(name, clst)
 
+
 def get_cmap(name):
     """Get a color map array.  May raise a KeyError if a map of the given name
     does not exist.
     """
     return cmaps[name]
 
+
 def get_names():
+    """Get colormap names."""
     res = list(cmaps.keys())
     res = sorted(res, key=lambda s: s.lower())
     return res
 
+
 def matplotlib_to_ginga_cmap(cm, name=None):
+    """Convert matplotlib colormap to Ginga's."""
     if name is None:
         name = cm.name
     arr = cm(numpy.arange(min_cmap_len))
     clst = tuple(map(lambda rec: tuple(rec)[:3], arr))
     return ColorMap(name, clst)
 
+
 def add_matplotlib_cmap(cm, name=None):
+    """Add a matplotlib colormap."""
     global cmaps
     cmap = matplotlib_to_ginga_cmap(cm, name=name)
     cmaps[cmap.name] = cmap
 
+
 def add_matplotlib_cmaps():
+    """Add all matplotlib colormaps."""
     import matplotlib.pyplot as plt
 
-    for name in plt.cm.datad.keys():
-        if not isinstance(name, str):
+    names = list(plt.cm.datad.keys())
+
+    # New default colormap for matplotlib 2.0, for those who cannot wait.
+    # See http://matplotlib.org/style_changes.html
+    if 'viridis' not in names:
+        names.append('viridis')
+        names.sort()
```

Comment:
I'm fine with leaving the line in, but maybe we should add a comment to the effect that the line is there because the colormap is somehow not picked up automatically with "transition" versions of matplotlib (or something to that effect).